java/test/jmri/jmrix/loconet/messageinterp/LocoNetMessageInterpretTest.java

```diff
@@ -6777,6 +6777,14 @@ public void testExpendedAccyReads() {
         Assert.assertEquals("read 6 Ext Accy Addr 8 CV 28", ""
                 + "Extended Accessory Decoder CV Verify: Address 8 CV 28, check if it is 0.\n",
                 LocoNetMessageInterpret.interpretMessage(l, "LT", "LS", "LR"));
+        
+        // [ED 0B 7F 54 17 3C 6E 65 03 5E 4F] Extended Accessory Decoder CV 'Verify': Address 500 CV 260, check if it is 222.
+        l = new LocoNetMessage(new int[] {0xED, 0x0B, 0x7F, 0x54,
+            0x17, 0x3C, 0x6E, 0x65, 0x03, 0x5E, 0x4F});
+
+        Assert.assertEquals("read 7 Ext Accy Addr 500 CV 260", ""
+                + "Extended Accessory Decoder CV Verify: Address 500 CV 260, check if it is 222.\n",
+                LocoNetMessageInterpret.interpretMessage(l, "LT", "LS", "LR"));
     }
```

Comment:
It would be better if the code would attempt to check a few different CV numbers, _especially_ CV numbers that exercise the CV numbers that were changed.  At a minimum, the tests should exercise CV Numbers 256, 257, 512, 513, and 1024, to verify bits 8 and 9, and the maximum CV number value.